netket/graph/hypercube.py

```diff
@@ -0,0 +1,54 @@
+from custom_graph import PyCustomGraph
+
+import numpy as _np
+import networkx as _nx
+
+class PyHypercube(PyCustomGraph):
+	r"""A hypercube lattice of side L in d dimensions. 
+	Periodic boundary conditions can also be imposed"""
+
+	def __init__(self, length, n_dim=1, pbc=False):
```

Comment:
I think that default value used to be pbc=True, I would keep that for consistency